index.js

```diff
@@ -124,4 +126,22 @@ if (platform === 'linux') {
 	main.questionMarkPrefix = '?';
 }
 
-module.exports = platform === 'win32' ? win : main;
+var figures = platform === 'win32' ? win : main;
+
+var fn = function (str) {
+	if (figures === main) {
+		return str;
+	}
+
+	Object.keys(main).forEach(function (key) {
+		if (main[key] === figures[key]) {
+			return;
+		}
+
+		str = str.replace(new RegExp(escapeStringRegexp(main[key]), 'g'), figures[key]);
```

Comment:
The `m` flag makes the regex work on a line-by-line basis if you use `^` and `$`, but we only replace one character globally, so it doesn't make a difference.